doc/doxygen/Makefile

```
@@ -20,6 +31,13 @@ html: src/changelog.md src/coc.md src/governance.md
 	( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
 	@echo ""
 	@echo "RIOT documentation successfully generated at file://$(RIOTBASE)/doc/doxygen/html/index.html"
+	@DOXYGEN_VERSION=$$(doxygen --version | awk '{print $$1}'); \
```

```suggestion
	@DOXYGEN_VERSION=$$(doxygen --version); \
```

should also do it, right?

or even define it above as Makefile variable as in `DOXYGEN_VERSION = $(shell doxygen --version)`

---

The first `awk` is required because the Doxygen version number can include the commit hash as well:
```
$ doxygen --version
1.12.0 (c73f5d30f9e8b1df5ba15a1d064ff2067cbb8267)
```

It still compares successfully, but the output is a bit ugly: `Warning: Doxygen version 1.12.0 (c73f5d30f9e8b1df5ba15a1d064ff2067cbb8267) is too old. It is recommended to use at least version 1.14.0 to avoid incorrectly formatted output.`

But I like the Makefile variable idea.

---

I can use the `cut` command as well, sure. `awk` is used in other places as well, so it's not introducing a new dependency, but perhaps the `cut` command is more obvious.